ddtrace/internal/debug.py

```diff
@@ -137,3 +137,82 @@ def collect(tracer):
         partial_flush_enabled=tracer._partial_flush_enabled,
         partial_flush_min_spans=tracer._partial_flush_min_spans,
     )
+
+
+def pretty_collect(tracer):
+    class bcolors:
+        HEADER = "\033[95m"
+        OKBLUE = "\033[94m"
+        OKCYAN = "\033[96m"
+        OKGREEN = "\033[92m"
+        WARNING = "\033[93m"
+        FAIL = "\033[91m"
+        ENDC = "\033[0m"
+        BOLD = "\033[1m"
```

Comment:
I wonder if these will work in all environments or not, are there some cases when we'll print the literals `\033[95m`, etc ?

Reply:
Yes, but it's not so messy that it makes it unreadable. Happy to change it in some way like printing with no colors, or adding a flag for no-color.

Reply:
a no color option could be nice to have, I'd be willing to wait and see what others think as well

Reply:
added colorless option